Scenes/Main.gd

```diff
@@ -12,4 +12,4 @@ func _on_ScrollingBackground_transition_updated(completionPercentage, _moonXPos)
 		
 		
 func _on_MainButtons_startGameClicked():
-	print("TODO: Go to to game scene!")
+	get_node("/root/SceneManager").goto_scene("res://Scenes/Gameplay/Level1/Level1.tscn")
```

Comment:
Great! Little advice: it should be possible to call `SceneManager` directly (without `get_node()`)!

Reply:
Good call. I tried it out and it worked with no problems.
https://github.com/crystal-bit/space-shooter/pull/35